utils/dashboard-helper.ts

```diff
@@ -62,7 +78,7 @@ export const runHelper = async (
     return false;
   }
 
-  const warnings: string[] = [];
+  let warnings: string[] = [];
```

Comment:
We can still use `const` here since we are not replacing the reference, just adding entries with `push` as before